examples/webgl_loader_viewer.html

```diff
@@ -0,0 +1,410 @@
+		loader.load( objPath , function ( myScene ) {
+
+			debugPrint(objPath+"<br>");
+			OnLoadScene(myScene, fileType, fileName);
+			resetCamera();
+
+		}, onProgress, onError );
+
+		//to do: improvement - post loader.load() fixes - handle camera for these files automaticaly
+		if (fileName==="mascot.tjs.sea")
+		{
+			camera.position.z = 2000;
+		}
+		else if (fileName==="house.wrl")
+		{
+			camera.position.z = 30;
+		}
+		else if (fileName==="kawada-hironx.dae"){
+			camera.position.z = 3;
+		}
+		else if (fileName==="duck.gltf"){
+			camera.position.z = 6;
+		}
+		else if (fileName==="Zaghetto.pcd")
+		{
+			//to do: improvement - Zaghetto.pcd need to flip pcd model upside down as needed
+			//to do: improvement - Zaghetto.pcd find correct camera settings
+		}
+	}
+
+	function init() {
+
+		camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 1, 2000 );
+		//camera.position.z = 100;
+		camera.position.z = 250;
+
+		//controls = new THREE.TrackballControls( camera );
+
+		// scene
+
+		scene = new THREE.Scene();
+
+		// texture
+
+		var texture = new THREE.Texture();
+
+		var material = new THREE.MeshBasicMaterial( { color: 'red' } );
+
+		// model
+		var model = 'models/babylon/skull.babylon';
+		loadModel(model);
+
+		//
+
+		renderer = new THREE.WebGLRenderer();
+		renderer.setPixelRatio( window.devicePixelRatio );
+		renderer.setSize( window.innerWidth, window.innerHeight );
+		document.body.appendChild( renderer.domElement );
+
+		//
+
+		controls = new THREE.OrbitControls( camera, renderer.domElement );
+
+		window.addEventListener( 'resize', onWindowResize, false );
+
+	}
+
+	function OnLoadScene(fileScene, objType, objName)
+	{
+
+		if (objType === "dae" || objType === "gltf" || objType === "sea")
+		{
+			fileScene = fileScene.scene;
+			//scene.scale.x = scene.scale.y = scene.scale.z = 0.05;
+		}
+
+		debugPrint("root children:" + fileScene.children.length+"<br>");
+		var space ="";
+		var childcnt = 0;
+		fileScene.traverse( function ( object ) {
+
+			//to do: bug - fileScene.traverse() dae error on traversing last child?
+
+			//to do: improvement - fileScene.traverse() handle nested groups?
+			if ( object.constructor.name === "Group" ) {
+				scene.add(object);
+				childcnt = object.children.length;
+				debugPrint("constructor: "+object.constructor.name+", children:" + childcnt+"<br>");
+				if (childcnt > 0)
+				{
+					space = "++";
+				}
+			}
+			else if ( object.constructor.name === "SkinnedMesh" || object.constructor.name === "Mesh" || object.constructor.name === "Object3D" ) {
+				if (childcnt > 0)
+				{
+					debugPrint(space+"child:" + childcnt+", ");
+				}
+				debugPrint("constructor: "+object.constructor.name+", children: "+object.children.length+"<br>");
+
+				if (objName === "skull.babylon" || objName === "duck.gltf" )
+				{
+					//to do: bug - with gltf file can not move model unless material is changed
```

Comment:
Finally fixed gltf camera, controls issue, apparently this is required: `THREE.GLTFLoader.Shaders.update(scene, camera)`
as shown below:

```
function animate() {

requestAnimationFrame( animate );
if (fileType === "gltf")
{
THREE.GLTFLoader.Shaders.update(scene, camera);
}
render();

}
```